docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/fractribution/index.md

```diff
@@ -89,6 +95,37 @@ Execute the following either through your CLI, within dbt Cloud, or within [Snow
 dbt run --select snowplow_fractribution
```

Comment:
Might be good to also notify the user here that they can run with Snowpark, after "Execute the following..." Maybe something like: "If you are using Snowflake and wish to automatically run the Python script in Snowpark, please first see `Run on Snowflake using Snowpark` below." to save them an extra step